notebooks/data-sources/TestGrid/testgrid_metadata_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
      "name": "stdout",
```

Comment:
<p>What about the data in tests?</p><p> </p><p>In addition to "statuses" there is also "linked_bugs", "messages", "short_texts", "user_property", "target", "original-name",</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='168'/>